src/exml.erl

```diff
@@ -39,23 +34,23 @@ xml_size([]) ->
     0;
 xml_size([Elem | Rest]) ->
     xml_size(Elem) + xml_size(Rest);
-xml_size(#xmlcdata{ content = Content }) ->
+xml_size(#xmlcdata{content = Content}) ->
     iolist_size(exml_nif:escape_cdata(Content));
-xml_size(#xmlel{ name = Name, attrs = Attrs, children = [] }) ->
+xml_size(#xmlel{name = Name, attrs = Attrs, children = []}) ->
     3 % Self-closing: </>
-    + byte_size(Name) + xml_size(Attrs);
-xml_size(#xmlel{ name = Name, attrs = Attrs, children = Children }) ->
+        + byte_size(Name)
+        + xml_size(Attrs);
+xml_size(#xmlel{name = Name, attrs = Attrs, children = Children}) ->
     % Opening and closing: <></>
-    5 + byte_size(Name)*2
-    + xml_size(Attrs) + xml_size(Children);
-xml_size(#xmlstreamstart{ name = Name, attrs = Attrs }) ->
+    5 + byte_size(Name) * 2 + xml_size(Attrs) + xml_size(Children);
+xml_size(#xmlstreamstart{name = Name, attrs = Attrs}) ->
     byte_size(Name) + 2 + xml_size(Attrs);
-xml_size(#xmlstreamend{ name = Name }) ->
+xml_size(#xmlstreamend{name = Name}) ->
     byte_size(Name) + 3;
 xml_size({Key, Value}) ->
-    byte_size(Key)
-    + 4 % ="" and whitespace before
-    + byte_size(Value).
+    byte_size(Key) +
+        4 % ="" and whitespace before
+        + byte_size(Value).
```

Comment:
Why here the `+` is at the begging while in the line it was moved to the line above? Is it because of the comment in the prev line?

Reply:
It's because of the comment, indeed.